main.cpp

```diff
@@ -506,19 +661,31 @@ struct Drone
     Preset presetA;
     Preset presetLand;
 
+    Drone();
+
     void liftOff();
     void maneuver(float input); 
     bool land();
 };
 
+Drone::Drone()
+{
+    std::cout << "Drone UDT being constructed!" << std::endl; //1)
+}
+
 void Drone::liftOff()
 {
-    if( gpsA.altitude > 0) // hmm
+    if( gpsA.altitude > 0.0f) std::cout << "Already airborn!\n";
+    else
     {
         propellerA.accelerate(3.0f);
         propellerB.accelerate(3.0f);
         propellerC.accelerate(3.0f);
         propellerD.accelerate(3.0f);
+
+        gpsA.altitude += 1.0f;
+
+        std::cout << "Lifting off!\n";
     }
```

Comment:
If you use curly braces for either `if` or `else`, use curly braces for both
- [ ] revise